azurerm/resource_arm_virtual_hub.go

```diff
@@ -106,6 +107,7 @@ func resourceArmVirtualHub() *schema.Resource {
 			},
 
 			"route": {
+				// TODO: confirm this wants to be a TypeSet
 				Type:     schema.TypeList,
```

Comment:
No index field here so that i think this field as list should be ok, right?

Reply:
no, in testing this needs to be a set